packages/prop-house-webapp/src/components/RoundContent/index.tsx

```diff
@@ -73,13 +73,7 @@ const RoundContent: React.FC<{
       dispatch(setActiveProposals(proposals));
 
       // default sorting method is random, unless the auction is over, in which case its by votes
-      dispatchSortProposals(
-        dispatch,
-        auctionStatus(auction) === AuctionStatus.AuctionEnded
-          ? SortType.VoteCount
-          : SortType.Random,
-        false,
-      );
+      dispatchSortProposals(dispatch, SortType.VoteCount, false);
```

Comment:
why default to Voting only/always here? vs in `Round` where you do [this](https://github.com/Prop-House/prop-house-monorepo/pull/339/commits/7742982e86a167e48ec663774d6afdabfbcbb318#diff-0a60086f6de602791af42754d450e8cf980aed8acd66aca8ff1b068d4f3c708e):
```
        auctionStatus(round) === AuctionStatus.AuctionEnded ||
          auctionStatus(round) === AuctionStatus.AuctionVoting
          ? SortType.VoteCount
          : SortType.Random,
```